var/spack/repos/builtin/packages/py-atlas-building-tools/package.py

```diff
@@ -11,31 +11,31 @@ class PyAtlasBuildingTools(PythonPackage):
     homepage = "https://bbpcode.epfl.ch/browse/code/nse/atlas-building-tools/tree/"
     git      = "ssh://bbpcode.epfl.ch/nse/atlas-building-tools"
 
-    version('0.1.0', commit='4d2ca679e34d08c76f96f15ccd3cdeccd2044696')
+    version('0.1.1', tag='atlas_building_tools-v0.1.1')
```

Comment:
Not for this PR, but it seems that also `atlas_building_tools` uses `_` instead of `-` as separator for `name` in `setup.py` (as it was for `cgal-pybind`).

Reply:
@GianlucaFicarelli  Thanks for all your comments and corrections.
Regarding the name used in setup.py, we have indeed the same problem as before.

I am in the process of writing a new commit in atlas-building-tools. The first thing I did was to change the name in setup.py, using dashes instead of underscores.

So it will be fixed soon.